src/main/java/com/iota/iri/Iota.java

```diff
@@ -113,6 +117,7 @@ public void init() throws Exception {
         udpReceiver.init();
         replicator.init();
         node.init();
+        databaseRecycler.init(new Date());
```

Comment:
```suggestion
        databaseRecycler.init(new Date(System.currentTimeMillis()));
```
for clarity?

this is what it does internally:
```
    public Date() {
        this(System.currentTimeMillis());
    }
```